docs/migration/5_0-checkout.md

```diff
@@ -1,9 +1,21 @@
 ## Checkout
 
-TODO:#checkout - write some introduction here, explaining the motive behind the refactor:
-  - subscribe to command and get the result in the same method call, instead of using three actions in ngrx
-  - the same is for the queries - subscribe to a query, and get the state feedback in the same call (loading, error or the data itself).
-  - easier to extend - everything is in classes, and it's easy to extends / override a method, as opposed to ngrx, where you can't really extend a reducer or an effect
+Checkout refactoring + code splitting was done as part of the initiative of removing `NGRX` dependencies from Spartacus by replacing it with [commands and queries](https://sap.github.io/spartacus-docs/commands-and-queries/#page-title).
+
+As we want to reduce the bundle size in Spartacus, the checkout refactoring also consisted of decoupling the checkout logic into three different entry points: base, b2b, scheduled-replenishment. 
```

Comment:
Lets just put colon here, as the unordered list on line 7 lists what are the entry points.

```suggestion
As we want to reduce the bundle size in Spartacus, the checkout refactoring also consisted of decoupling the checkout logic into three different entry points:
```

Reply:
done 100%